src/main/java/frc/robot/vision/AprilTagCamera.java

```diff
@@ -367,7 +367,7 @@ private void updateEstimationStdDevs(
         avgDist /= numTags;
         // Decrease std devs if multiple targets are visible
         if (estimatedPose.get().strategy == PoseStrategy.MULTI_TAG_PNP_ON_COPROCESSOR) {
-          estStdDevs = multiTagStdDevs;
+          // estStdDevs = multiTagStdDevs;

```

Comment:
I don't remember why I took this out, since we aren't multitag, it doesn't run anymore, but you might want to put it back...